web_widget_image_download/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
@legalsylvain, he is a new guy in our team (Tecnativa), so we are deciding publicly which license to use.